### PR TITLE
Use docker-like names for tasks

### PR DESCRIPTION
**Pull request by karimodm**

### CODE REVIEW

File: commands/eureka-run.js [hunk 1]
```diff
@@ -20,17 +21,20 @@ program
     command = cmd.join(' ')
   })
   .option('-t, --tier [tier]', 'Run on machine of type [tier]', 'n1-standard-1')
+  .option('-n, --name [name]', 'Assign a name to the task', names.getRandomName().replace(/_/g, '-'))
```
Comment (alongalky) [inline commands/eureka-run.js:24]: Why change the underscores?

File: commands/eureka-run.js [hunk 2]
```diff
@@ -20,17 +21,20 @@ program
...
 
   .parse(process.argv)
 
 if (!machineName || !command) {
   program.help()
+} else if (!program.name.match(/^[a-z0-9][-a-z0-9]{0,50}[a-z0-9]$/)) {
+  console.error('Invalid task name, only lowercase alphanumeric characters allowed')
```
Comment (alongalky) [inline commands/eureka-run.js]: Please use printError for consistency (or at least console log with color instead of console.error)

**Review by alongalky — CHANGES_REQUESTED** [review]

File: commands/eureka-run.js (hunk 1 above)
Comment (karimodm) [reply commands/eureka-run.js:24]: Google does not support _ in tags

File: commands/eureka-run.js (hunk 2 above)
Comment (karimodm) [reply commands/eureka-run.js]: I wanted to, but there is no such thing as printing a custom error message with printError.
I will use chalk for this.

File: errors/print-error.js [hunk 3]
```diff
@@ -21,5 +21,5 @@ module.exports = err => {
       err.toString()
   }
 
-  console.log(chalk.red('ERROR'), message)
+  console.error(chalk.red('ERROR'), message)
```
Comment (alongalky) [inline errors/print-error.js:27]: Please use printError

**Review by alongalky — CHANGES_REQUESTED** [review]

File: errors/print-error.js (hunk 3 above)
Comment (karimodm) [reply errors/print-error.js:27]: We are in it...

**Review by alongalky — APPROVED** [review]

**Pull request merged**